development/image-syncer/external-images.yaml

```diff
@@ -14,8 +14,12 @@ images:
 - source: "grafana/loki:2.2.1"
 - source: "grafana/grafana-image-renderer:3.2.1"
 - source: "hudymi/mockice:0.1.3"
-- source: "istio/pilot:1.11.4-distroless"
-- source: "istio/proxyv2:1.11.4-distroless"
+- source: "istio/pilot@sha256:a7c55cd2e84ecb4f3d3ac4f62468247363ddde1b0cbb6aeeda967e2691c30b30"
+  tag: "1.11.4-distroless"
```

Comment:
Can you add something, at the end? Most images use `-v1`; this is the tag that the image will be available at on our side, we want to have different name to not overwrite existing image